index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./src/styles.css">
+    <title>Document</title>
+    <h1>About Me</h1>
+</head>
+<body>
+        <P>Interested in merging poetry and photography</P>
```

Comment:
this p tag should not be capitalized 

Also a word on indentation. If a tag is a child of another tag then it is indented one time. If it is a sibling then it is at the same level (indent). So your p, img, ul, and ol tags should all be indented the same as the h2 tags, while your li tags are children of the ul and ol tags so they should be indented 2 times.